rules/windows/process_creation/win_renamed_binary_highly_relevant.yml

```diff
@@ -17,19 +17,19 @@ logsource:
 detection:
     selection:
         OriginalFileName:
-            - "powershell.exe"
-            - "powershell_ise.exe"
-            - "psexec.exe"
-            - "psexec.c"  # old versions of psexec (2016 seen)
-            - "cscript.exe"
-            - "wscript.exe"
-            - "mshta.exe"
-            - "regsvr32.exe"
-            - "wmic.exe"
-            - "certutil.exe"
-            - "rundll32.exe"
-            - "cmstp.exe"
-            - "msiexec.exe"
+            - "*powershell.exe"
```

Comment:
According to the referenced articles, the original file name does only include the filename itself - so no wildcard is needed.

Reply:
Just read your commit message - should the ES backend be changed to mitigate this behavior?

Reply:
I just revoked my changes on this file, as it was not the right solution - it was more of a temporary bypass to indeed solve an issue we have been facing in using the es-dsl backend: wildcard queries are the only way we currently see fit in Sigma to haven an exact match on a field, instead of using a phrase query. Hence why we sometimes add a wildcard to force sigma to use wildcard queries, which are much more predictable than the phrase queries (I have lots of other examples across other use cases that cause FP's, such as the User Agent ones).

I reverted my changes as I indeed think that this deserves changes to the es-dsl backend, all my other commits in this PR are still relevant! Appreciate your review. Will create a ticket for the es-dsl backend change I would propose.